app/src/main/res/layout/activity_main.xml

```diff
@@ -63,21 +63,23 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_marginBottom="12dp"
+                    android:textIsSelectable="true"
```

Comment:
Why this change?

Reply:
**This only affects the sample app.** It makes the text selectable so one can easily copy the device name / fcm token and use it on the firebase console to send test fcm notifications . No cosmetic change.